src/main/java/com/openshift/restclient/authorization/IAuthorizationContext.java

```diff
@@ -26,7 +26,7 @@
 	
 	/**
 	 * 
-	 * @return true if the token is non-null; false otherwise
+	 * @return true if authorized; false otherwise
```

Comment:
the impl now seems to do remote calls which should at least be mentioned here. But I believe that we should not do remote calls in this metod since it's a change in semantics which I'd only do if really really needed..